src/review.ts

```diff
@@ -0,0 +1,201 @@
+import {
+  html,
+  css,
+  LitElement,
+  nothing,
+  HTMLTemplateResult,
+  CSSResultGroup,
+} from 'lit';
+import { property, customElement, state } from 'lit/decorators.js';
+import { msg } from '@lit/localize';
+
+import { Review } from '@internetarchive/metadata-service';
+
+import starBasic from './assets/star-basic';
+
+/* Further properties for reviews added before render */
+export interface ReviewForRender extends Review {
+  screenname: string;
+  domId: string;
+  itemname?: string;
+}
+
+/**
+ * Renders a single IA review
+ */
+@customElement('ia-review')
+export class IaReview extends LitElement {
+  /* The review to be rendered */
+  @property({ type: Object }) review?: ReviewForRender;
+
+  /* Maximum length for subject */
+  @property({ type: Number }) maxSubjectLength = 100;
+
+  /* Maximum length for body */
+  @property({ type: Number }) maxBodyLength = 1000;
+
+  /* Base for URLs */
+  @property({ type: String }) baseHost = 'https://archive.org';
+
+  /* Whether to show truncated review title / body */
+  @state()
+  private showTruncatedContent: boolean = false;
+
+  render() {
+    return !this.review
+      ? html`<div class="error">Could not render review.</div>`
```

Comment:
maybe a less techy way to say?
```suggestion
      ? html`<div class="error">Review cannot be displayed at this time.</div>`
```

Reply:
Good thinking! Will also check in with Jim re: phrasing